osutil/cp.go

```diff
@@ -63,7 +64,14 @@ func CopyFile(src, dst string, flags CopyFlag) (err error) {
 		// Our native copy code does not preserve all attributes
 		// (yet). If the user needs this functionatlity we just
 		// fallback to use the system's "cp" binary to do the copy.
-		return runCpPreserveAll(src, dst)
+		err := runCpPreserveAll(src, dst, "copy all")
```

Comment:
This  and the following line could be a sinle line: `if err := runCp...; err != nil`